src/fileformats/ocd_file_export.cpp

```diff
@@ -23,6 +23,7 @@
 #include "ocd_file_export.h"
```

Comment:
If there were a way to comment on the patch as whole...

This patch should be squashed into the previous one. See also additional comments below.

Reply:
Probably you mean the commit which introduces the icon types?

Reply:
`a4a96070002e5 Ocd: Move icon palette and size to format headers`

I commented on individual patches. Switching to "Commits" and looking at the patches one by one will put the comments into context. For the next time I'll try to make my commentary compatible with the "merged code change" view.